R/getCrossAssociation.R

```diff
@@ -731,8 +734,8 @@ setMethod("getCrossAssociation", signature = "SummarizedExperiment",
 
 ########################### .check_that_assays_match ###########################
 # If correlations between features are analyzed, samples should match, and vice versa
-.check_that_assays_match <- function(assay1, assay2, MARGIN){
```

Comment:
Revert changes in this iterrnal function

Reply:
ACK.